### ISSUE TITLE
Unable to boot to install - i5 Matebook X Pro 2019 MX250

### ISSUE THREAD

**Issue by mtimms1980** [gnodipac886/MatebookXPro-hackintosh#113]
Hi, I'd very much appreciate any help anyone can offer. I've been trying to get my i5 X Pro to boot and Install using this method, but the best I have managed so far is the Apple logo with progress bar slowing increasing, but never proceeding further than that screen. If I boot with -v I can see the attached, which appears to just continuously cycle through the same text.

I've tried creating the usb using the GibMacOS Windows creation method, and using a macOS VM but get similar results. I've copied the clover files from this repo to my USB drive but if I am honest, I am not 100% sure what exactly I am meant to copy, and to where.

Thanks for all of the hard work and support on this project, I can't wait to get macOS up and running on this awesome laptop! 
![IMG_20200525_091804](https://user-images.githubusercontent.com/44544927/82794703-d37b0c00-9e6a-11ea-87e8-69323269243e.jpg)

**Comment by AshleyMiller9696** [comment]
i used this offline tutorial https://kb.hackintoshisfun.ml/clover/prerequisites/get-started/offline-installer-prerequisites
for the kexts i used the ones in the "Clover" Folder - so when it comes to installation the touchpad should work
in clover i put -disablegfxfirmware as an boot argument

**Comment by mtimms1980** [comment]
> i used this offline tutorial https://kb.hackintoshisfun.ml/clover/prerequisites/get-started/offline-installer-prerequisites
> for the kexts i used the ones in the "Clover" Folder - so when it comes to installation the touchpad should work
> in clover i put -disablegfxfirmware as an boot argument

Thanks for the reply, I will give that a try today.

So to confirm, you used the guide in the link you provided to create the installer and then used the kexts from the CLOVER folder from this repo? Did you use the kexts from the "\USB-EFI\CLOVER" folder or from the "\CLOVER" folder?

**Comment by AshleyMiller9696** [comment]
correct -> just copied the files form the \Clover folder

**Comment by mtimms1980** [comment]
Alright, I'll give it a go. You have the X Pro i5 8265u?

**Comment by profzei** [comment]
In case of some more troubles, [here](https://github.com/profzei/Matebook-X-Pro-2018) you can find an updated repo for our machine!

**Comment by mtimms1980** [comment]
> In case of some more troubles, [here](https://github.com/profzei/Matebook-X-Pro-2018) you can find an updated repo for our machine!

I just came across your repo! But I think what I have is a 2019 model... i5 8265u CPU. Doesn't seem to matter what I try, I cannot get it to boot to an installer!

**Comment by AshleyMiller9696** [comment]
Yes I also have the i5, but the PowerManagment files in this repo are fine to get it to work

2018+2019 should be the same since the only difference is the graphics card - which is disabled in macOS

**Comment by mtimms1980** [comment]
Ok I'll give it another go. Can't seem to get the combinations right though, different configurations do seem to produce different results, but no successful boot yet.

**Comment by AshleyMiller9696** [comment]
+ don‘t know if 10.15.5 works
-> I’m still on 10.15.4

So Clover works & you get to the MacOs installer, but when installing it takes forever?

**Comment by profzei** [comment]
> don‘t know if 10.15.5 works

I updated yesterday to 10.15.5 without any issue...

**Comment by AshleyMiller9696** [comment]
> > don‘t know if 10.15.5 works
> 
> I updated yesterday to 10.15.5 without any issue...

I got an Kernel-Cache Error ://

**Comment by mtimms1980** [comment]
> * don‘t know if 10.15.5 works
>   -> I’m still on 10.15.4
> 
> So Clover works & you get to the MacOs installer, but when installing it takes forever?

I don't get past first boot, without -v I see the Apple logo and progress bar, which slowly increases but gets to the end and doesn't progress last that screen. If I boot with -v I see that it looks like a never ending loop of acpi errors as per the screen shot.

**Comment by mtimms1980** [comment]
No I haven't installed anything yet. I have tried numerous combinations of drivers and kexts but I cannot get to the point of installing macOS.

**Comment by mtimms1980** [comment]
Despite trying what feels like every possible combination of everything (not ready if course!), I still cannot get passed initial boot of the USB install.

I'm still not sure if anyone has confirmed they have managed to boot and install on a 2019 X Pro i5 8265u MX250. I am on BIOS rev 1.19 which I believe is the latest for this model.

Using https://github.com/profzei/Matebook-X-Pro-2018 from profzei I am get stuck on "ApplePS2Controller: notification consumer published: ApplePS2Mouse". Researching this now.

**Comment by mtimms1980** [comment]
> Please try this dedicated repo for the 2019 model: https://github.com/laozhiang/MateBook_13_14_XPro-Hackintosh
> 
> if you don't understand Chinese, just use google translate

This! This worked like a charm, thank you for pointing me in this direction, and credit to @laozhiang

Just need to experiment with kexts to fine tune a bit, and I think i need a custom/specific CPUFriend... Not sure sleep is working but happy to experiment now that it boots. Thanks you.

**Comment by mc6415** [comment]
@mtimms1980 I was wondering if you had any luck with this? I've got it all installed but sound just does not work at all for me right now.

**Comment by gnodipac886** [comment]
I think that sound should be the same model. Just make sure that appleALC is installed and you can use layout 97 which is the one I worked on.

**Comment by mtimms1980** [comment]
> @mtimms1980 I was wondering if you had any luck with this? I've got it all installed but sound just does not work at all for me right now.

Sound wasn't an issue for me, using the files in the following folder: https://github.com/laozhiang/MateBook_13_14_XPro-Hackintosh/blob/master/install/EFI_for_MateBook_X_Pro_2019.zip

Battery drain seems a bit high, and WiFi is hit and miss using the Intel WiFi still amazing that WiFi works at all! I haven't got around to trying additional kexts (CPUFriend mainly) but plan to ASAP.

**Comment by Chilluminati91** [comment]
Thank god i checked this issues section. Same error as OP, related to battery management on the Matebook X Pro 2020. Will check the chinese repo

**Comment by alexbiship** [comment]
@Chilluminati91  I've got a new matebook x pro 2020 model, i7 10th intel CPU. I tried to install Catalina with various of methods but didn't work at all. anyone can help me? Thanks.
